### ISSUE TITLE
Please update the SDK to support .NET Standard 2.0, .NET Standard 2.1, and .NET Core 3.1

### ISSUE THREAD

**Issue by eriawan** [OfficeDev/Open-XML-SDK#699]
# Before submitting an issue, please fill this out

Is this a:
- [ ] Issue with the OpenXml library
- [x] Question/Proposal on library usage

**Description**
Please update the SDK to support .NET Standard 2.0, .NET Standard 2.1, and .NET Core 3.1, because current library only support .NET Standard before .NET Standard 2.0. 
These .NET Standard before 1.6 needs more API surface referenced as nuget packages. Therefore We need to add supports for .NET Standard 2.0, .NET Standard 2.1 and hopefully .NET Core 3.1 as well, as .NET Core 3.0 has reached end of support.
**Information**

- .NET Target: (ie .NET Framework, .NET Core, UWP, Xamarin, etc)
- DocumentFormat.OpenXml Version: (ie 2.7.2)

**Repro**

```csharp
// Please add a self-contained, minimum viable repro of the issue.
// If you require external resources, please provide a gist or GitHub repro
// An Xunit style test is preferred, but a console application would work too.
``` 

**Observed**

N/A, this is a proposal to update SDK to support .NET Standard 2.0, .NET Standard 2.1, and .NET Core 3.1

**Expected**

Open XML SDK support .NET Standard 2.0, .NET Standard 2.1, and .NET Core 3.1

**Comment by lindexi** [comment]
@eriawan The repo can support .NET Standard 1.3 that means it support .NET Standard 2.0 and so on.

**Comment by twsouthwick** [comment]
.NET Standard 2.0 support was added in 2.10 (updated to 2.10.1 as there were some regressions in 2.10 that were fixed in 2.10).  However, the target a library is compiled against is an orthogonal decision as to where it will run. We currently target the following:

- .NET 3.5
- .NET 4.0
- .NET 4.6
- .NET Standard 1.3
- .NET Standard 2.0

There are currently no plans at the moment for .NET Standard 2.1 nor .NET Core App 3.1 as those don't add anything we need and just increase library size and compilation time. We may target that at some point to use some of the Span<> related APIs, but we haven't needed that yet.

By the phrasing of your request, it seems you misunderstand support for .NET Standard. At this point, a lower .NET Standard can be supported anywhere a higher is supported; ie .NET Standard 1.3 works everywhere .NET Standard 2.0 does. There is a recommendation to support .NET Standard 2.0 *in addition to* .NET Standard 1.3 because of the number of auxiliary libraries brought in; for this reason we added support for .NET Standard 2.0. For more information, please see https://docs.microsoft.com/en-us/dotnet/standard/net-standard

iI'll close for now but feel free to reopen if you have other thoughts.

**Comment by eriawan** [comment]
Thanks @twsouthwick
